css/ace-mantis.css

```diff
@@ -214,25 +214,49 @@ pre {
     position: relative;
 }
 
-.ace-nav .nav-user-photo {
-    border-radius: 4px !important;
+.ace-nav .nav-user-photo-40-avatar-container {
```

Comment:
nav-user-photo is defined in other css:
- ace-mantis.css (2 matches)
- ace-rtl.css (4 matches)
- ace-rtl.min.css (4 matches)
- ace.css (4 matches)
- ace.min.css (4 matches)
Maybe it should stay as is.